qcodes/tests/dataset/legacy_DB_generation/generate_version_0.py

```diff
@@ -0,0 +1,54 @@
+# Generate version 0 database files for qcodes' test suite to consume
+
+import os
+
+from git import Repo
+
+# NB: it's important that we do not import anything from qcodes before we
+# do the git magic (which we do below), hence the relative import here
+import utils as utils
+
+gitrepopath = os.sep.join(os.path.realpath(__file__).split(os.sep)[:-5])
```

Comment:
Would it be useful to assert that this is actually a git repo e.g. check that there is a .git folder

Reply:
Or perhaps that is already handled below by Repo() ?

Reply:
As discussed offline, `Repo()` does that.